### ISSUE TITLE
Pres: server: specify private key doesn't work with PPK format

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#918]
We either need to figure out why this doesn't work or provide feedback to the user that PPKs won't work and to use `ssh-keygen`erated keys or suggest they use `ssh-add` in the commandline which will allow Komodo to use the key.

**Comment by Naatan** [comment]
I'm fairly certain the "why it doesn't work" part is simply because we do not support it. I'm not sure if we can verify whether the specified file is supported, ssh key files can have wildly different extension names.

**Comment by mitchell-as** [comment]
PPK files are PuTTY-formatted SSH keys. You need to take an existing private key and run it through PuTTY to get a PPK that can be loaded by Pageant (something like ssh-agent, but for Windows). I used to have to do this.

**Comment by cgchoffman** [comment]
> PPK files are PuTTY-formatted SSH keys.

yes, sorry I thought I had said that but I had only said it in the forum post: http://forum.komodoide.com/t/sftp-login-with-key-file/2317/17

**Comment by Naatan** [comment]
Right, well we could certainly "blacklist" certain extensions. What I'm saying is it would be hard to "whitelist" them. Although I suppose we could whitelist the most used ones and then allow for the user to change the file scope to "all".

**Comment by cgchoffman** [comment]
Is it not an option to look under the hook and make it work with PPK?  I don't see why the first option is to blacklist/whitelist files.

I would think it's possible unless you know something I don't @Naatan:
1. paramiko knows about PuTTY already
2. PuTTY supports making SSH connection where you explicitly specify a Key; ie. 'ssh -i path/to/key user@server`

**Comment by Naatan** [comment]
It's certainly possible, but given our current schedule it does not seem worth the time investment. I think step 1 would be to improve the UX. Step 2 (which would be long term) could be to support Putty key files.

**Comment by cgchoffman** [comment]
We should consider an easy fix like a message that says (this doesn't work for PuTTY keys) or something.  We can do a proper fix later.  @Naatan do mind if I bump this to 10.2 to see if I can fit it in during my bug fix phase?

**Comment by Naatan** [comment]
Assuming it's a small change; sure, but otherwise I'd prefer to keep this for 11.

I think simply showing more helpful "error" message could go a long way.

**Comment by cgchoffman** [comment]
> Assuming it's a small change;

I would think a message in text in the UI is a small change no?

**Comment by Naatan** [comment]
Yup I don't see why it wouldn't be.
